testflight/suite_test.go

```diff
@@ -133,6 +134,25 @@ var _ = AfterEach(func() {
 	fly("destroy-pipeline", "-n", "-p", pipelineName)
 })
 
+func downloadFly(atcUrl string) (string, error) {
+	client := concourse.NewClient(atcUrl, http.DefaultClient, false)
+	readCloser, _, err := client.GetCLIReader("amd64", runtime.GOOS)
+	if err != nil {
+		return "", err
+	}
+	outFile, err := ioutil.TempFile("", "fly")
```

Comment:
Sorry, missed this on my last review - we should probably handle the error here.

```suggestion
	outFile, err := ioutil.TempFile("", "fly")
	if err != nil {
		return "", err
	}
```

Reply:
oh, of course! don't sweat it. I feel like this PR has been on my backburner for its whole life haha